src/server/connection.rs

```diff
@@ -1815,10 +1817,12 @@ impl Connection {
             if crate::platform::is_installed()
                 && crate::platform::is_share_rdp()
                 && !*CONN_COUNT.lock().unwrap() > 1
+                && !self.checked_multiple_session
             {
                 if !self.handle_multiple_user_sessions(self.lr.clone()).await {
                     return false;
                 }
+                self.checked_multiple_session = true;
             }
```

Comment:
```
#[cfg(target_os = "windows")]
if !self.checked_multiple_session && crate::platform::is_installed()
    && crate::platform::is_share_rdp()
    && !*CONN_COUNT.lock().unwrap() > 1
{
    if !self.handle_multiple_user_sessions(self.lr.clone()).await {
        return false;
    }
}
self.checked_multiple_session = true;
```
 This will be called on every message, optimization will be worth it, and it only needs to be checked when the connection authorization has just succeeded. Does the code above works?
![1706776039583](https://github.com/rustdesk/rustdesk/assets/14891774/ef5d94b7-472e-4b3a-add0-84e607e7821e)